packages/ember-flight-icons/.prettierrc.cjs

```diff
@@ -1,19 +1,17 @@
-'use strict';
+"use strict";
 
 module.exports = {
-  singleQuote: true,
```

Comment:
@aklkv just to double check: now that we've reverted prettier to `2.8.8` is this change still OK?